### ISSUE TITLE
Item embed throwing error/not functional when adding item, clicking save.  

### ISSUE THREAD

**Issue by kelea99** [pulibrary/dpul#1239]
Item embed widget is not functional for MVW or for scanned resource selections I tried.  error screenshot below. What i did:
1. selected item embed widget
2. selected item within collection
3. there were no options to select a page within the item
4. clicked save.

![Screen Shot 2021-11-11 at 11 39 50 AM](https://user-images.githubusercontent.com/45043014/141335403-1ed40b23-8e16-484a-936b-24b6bedd52de.png)

Potentially related to #769

**Comment by kelea99** [comment]
Ok. Item embed for scanned resource I got to show, but it is not functioning as expected on front end.  I selected an image to be the thumbnail, and that is not functional.  I also cannot navigate throughout the item (it almost looks like all the images are sandwiched one on top of the other in the container, with the last image on top)
![Screen Shot 2021-11-11 at 12 56 52 PM](https://user-images.githubusercontent.com/45043014/141346447-e1fd761f-0a48-49f8-a7e2-8399a577e801.png)
